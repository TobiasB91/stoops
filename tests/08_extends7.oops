CLASS Main IS
    METHOD derived(a : A) IS BEGIN
    END METHOD

    METHOD main IS BEGIN
        derived(NEW Main);
    END METHOD
END CLASS

CLASS A EXTENDS Main IS
END CLASS