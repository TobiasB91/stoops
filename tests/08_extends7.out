Fehler in Zeile 6, Spalte 17: Ausdruck vom Typ A erwartet
